frontend/src/metabase-types/api/settings.ts

```diff
@@ -191,54 +191,62 @@ export interface OpenAiModel {
 export type HelpLinkSetting = "metabase" | "hidden" | "custom";
 
 interface InstanceSettings {
-  "active-users-count"?: number;
   "admin-email": string;
-  "bcc-enabled?": boolean;
-  "deprecation-notice-version"?: string;
-  "ee-openai-api-key"?: string;
-  "embedding-secret-key"?: string;
   "enable-embedding": boolean;
   "enable-nested-queries": boolean;
   "enable-query-caching"?: boolean;
-  "query-caching-ttl-ratio": number;
-  "query-caching-min-ttl": number;
   "enable-public-sharing": boolean;
   "enable-xrays": boolean;
-  "google-auth-auto-create-accounts-domain": string | null;
-  "google-auth-configured": boolean;
-  "jwt-enabled"?: boolean;
-  "jwt-configured"?: boolean;
-  "openai-api-key": string | null;
-  "openai-organization": string | null;
-  "openai-model": string | null;
-  "openai-available-models"?: OpenAiModel[];
-  "premium-embedding-token": string | null;
-  "saml-configured"?: boolean;
-  "saml-enabled"?: boolean;
   "search-typeahead-enabled": boolean;
-  "session-cookie-samesite": SessionCookieSameSite;
-  "show-database-syncing-modal": boolean;
   "show-homepage-data": boolean;
   "show-homepage-pin-message": boolean;
   "show-homepage-xrays": boolean;
   "site-uuid": string;
-  "slack-app-token": string | null;
-  "slack-files-channel": string | null;
-  "slack-token": string | null;
-  "slack-token-valid?": boolean;
   "subscription-allowed-domains": string | null;
-  "token-status": TokenStatus | null;
   "user-locale": string | null;
```

Comment:
I'm guessing this belongs to `UserSettings`.

Not sure about `user-visibility`.

Reply:
That's going to be the next PR.
We have a specific requirement for some of the settings called `:authenticated`. This one falls into that scope.